docker-compose.yml

```diff
@@ -11,5 +11,4 @@ services:
       MYSQL_PASSWORD: flab-bbt
       MYSQL_ROOT_PASSWORD: flab-bbt
     volumes:
-      - "./sql/:/docker-entrypoint-initdb.d"
-    command: --init-file /docker-entrypoint-initdb.d/schema.sql
\ No newline at end of file
+      - "./sql/:/docker-entrypoint-initdb.d"
```

Comment:
@jungin-jin-choi 
docker-compose.yml 위와 같이 수정하였습니다. docker-entrypoint-initdb.d 폴더에 sql 확장자가 달린 스크립트가 있으면 초기 컨테이너 구동시 해당 폴더 밑에 있는 모든 sql 스크립트를 알파벳순으로 1회 실행하게 됩니다. 굳이 command로 처리할 필요가 없습니다.

[mysql image 공식문서](https://hub.docker.com/_/mysql)  참조바랍니다.
> When a container is started for the first time, a new database with the specified name will be created and initialized with the provided configuration variables. Furthermore, it will execute files with extensions .sh, .sql and .sql.gz that are found in /docker-entrypoint-initdb.d. Files will be executed in alphabetical order. You can easily populate your mysql services by [mounting a SQL dump into that directory](https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-file-as-a-data-volume) and provide [custom images](https://docs.docker.com/reference/builder/) with contributed data. SQL files will be imported by default to the database specified by the MYSQL_DATABASE variable.